try {
    data.push(value);
    return a + b;
    alert('Action completed!');
} catch (error) {
    console.error("Error:", error);
}